cogs/guildOwners.py

```diff
@@ -323,74 +323,80 @@ async def register_error(self, interaction, error):
-    #         data = {
-    #             "guildId": ctx.guild.id,
-    #             "ownerId": ctx.author.id,
-    #             "mngRoleId": int(r.id),
-    #             "mngRoleName": f'{r}'
-    #         }
-    #
-    #         if self.backoffice.voting_manager.register_server_for_voting_service(data=data):
-    #             await customMessages.system_message(ctx=ctx, color_code=0,
-    #                                                 message=f'You have successfully activated ballot voting '
-    #                                                         f'pools functionality for your'
-    #                                                         f' server. Proceed with `{self.command_string}ballot` over '
-    #                                                         f'public channel where Crypto Link has access to.',
-    #                                                 destination=1, sys_msg_title=CONST_SYS_MSG)
-    #         else:
-    #             msg_title = ':ballot_box: __Ballot Voting System Error__ :ballot_box: '
-    #             message = f'System could not activate Ballot Voting pools due to backend issue. Please contact' \
-    #                       f' crypto link team.'
-    #             await customMessages.system_message(ctx=ctx, sys_msg_title=msg_title, message=message, color_code=0,
-    #                                                 destination=1)
-    #     else:
-    #         msg_title = ':ballot_box: __Ballot Voting System Already Active__ :ballot_box: '
-    #         message = f'You have already activated {ctx.guild} server for voting functionality.'
-    #         await customMessages.system_message(ctx=ctx, sys_msg_title=msg_title, message=message, color_code=0,
-    #                                             destination=1)
-    #
-    # @activate.error
-    # async def activate_error(self, ctx, error):
-    #     if isinstance(error, commands.CheckFailure):
-    #         message = f"You are required to specify active role on the server which will have access to the " \
-    #                   f"Ballot voting system management. `{self.command_string}owner ballot activate @discord.Role'"
-    #         await customMessages.system_message(ctx=ctx, color_code=1, message=message, destination=1,
-    #                                             sys_msg_title="Ballot voting access error!")
-    #     elif isinstance(error, commands.MissingRequiredArgument):
-    #         message = f"You forgot to provide @discord.Role which will have access to ballot functions."
-    #         await customMessages.system_message(ctx=ctx, color_code=1, message=message, destination=1,
-    #                                             sys_msg_title="Ballot voting access error!")
+    # ----------------Voting pools registration-------------#
+    @owner.subcommand(name="ballot", description="Crypto Link Ballot System")
```

Comment:
I think this needs to be a group and not subcommand in generator because, owner should do 
/owner ballot activate. in order to register the server for the ballots